### DISCUSSION TITLE
Back to the Future: 2014

### DISCUSSION THREAD

**Discussion by swvanderlaan** [Homebrew/discussions#2417] [Getting started] [answered]
Hi,
Long story short, I want to go back to the future of 2014. 
I will have a Mac system with Yosemite, XCode, Java 8, the works. 

Is it possible to also install brew dating back to around June 2014? And if so: how would I go about doing that? And how can I get packages installed from around that time?

Thanks

Sander

**Comment by SMillerDev** [comment, marked as answer]
It's all git, just check out a commit from around that time.

I'm not expecting it to work though.

**Reply by swvanderlaan** [reply]
Thanks!
